src/views/ManageList.jsx

```diff
@@ -1,9 +1,26 @@
 import { useState } from 'react';
-import { addItem } from '../api';
+import { addItem, shareList } from '../api';
 
-export function ManageList({ listPath }) {
+export function ManageList({ listPath, userId }) {
 	const [item, setItem] = useState({ name: '', urgency: 'soon' });
 	const [submitted, setSubmitted] = useState();
+	const [emailInvite, setEmailInvite] = useState('');
+	const [emailExists, setEmailExists] = useState();
+
+	const handleEmailInviteChange = (e) => {
+		setEmailInvite(e.target.value);
+	};
+
+	const handleEmailInviteSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			await shareList(listPath, userId, emailInvite);
+			setEmailExists('Your list was shared!');
```

Comment:
Minor nit: could reset the form value after the list has been shared.

Reply:
fixed!